src/site/_data/blogTags.js

```diff
@@ -0,0 +1,307 @@
+module.exports = {
+  "3d": {
+    tag: "3d",
+    title: "3D",
+    href: "/tags/3d/",
```

Comment:
yes 👍

Reply:
or, per the comment down in paged.njk, you might not even need it. paged.njk could just do `<a href="/tags/{{ tag }}">`

Reply:
For the reusability of the chips I did not want to do this. I thought it would be better to just pump in the href in it's full rather than have chips bake the href

Reply:
If I'm not mistaken, I think the href gets set by the page template or shotcode that's using the chip doesn't it? If that's the case then it might be ok for that parent to know that it's setting `/tags/{{tag}}`